Client/Frontend/Autofill/CreditCard/CreditCardBottomSheet/CreditCardBottomSheetViewController.swift

```diff
@@ -108,6 +108,7 @@ class CreditCardBottomSheetViewController: UIViewController, UITableViewDelegate
 
         self.viewModel.didUpdateCreditCard = { [weak self] in
             self?.cardTableView.reloadData()
+            self?.cardTableView.isScrollEnabled = self?.cardTableView.contentSize.height ?? 0 > self?.view.frame.height ?? 0
```

Comment:
I think this logic will only work in certain cases as it is not accounting for the height of the buttons which are also part of the view.

Reply:
the buttons are not part of the Used Saved Cards view